### ISSUE TITLE
no reply even though the flow rule is inserted

### ISSUE THREAD

**Issue by ARCHAJ** [abulanov/sfc_app#1]
Hello everyone,
I appreciate your efforts developping the application.
I would like to contribute to the application by implementing my proposal using the application.
as a first step, I am trying to test the application following exactly the readme file. Yet, I get no reply after inserting the flow. when I check the OVS I find the rule, but still the command tracepath gives no repy:
mininet> h1 tracepath h5
 1?: [LOCALHOST]                                         pmtu 1500
 1:  no reply
 2:  no reply
 3:  no reply
can someone guess why?

**Comment by abulanov** [comment]
Please elaborate on which step it failed, was it working before applying rules, some outputs showing symptoms is usually helpful for starting troubleshooting.

**Comment by abulanov** [comment]
> Hello everyone,
> I appreciate your efforts developping the application.
> I would like to contribute to the application by implementing my proposal using the application.
> as a first step, I am trying to test the application following exactly the readme file. Yet, I get no reply after inserting the flow. when I check the OVS I find the rule, but still the command tracepath gives no repy:
> mininet> h1 tracepath h5
> 1?: [LOCALHOST] pmtu 1500
> 1: no reply
> 2: no reply
> 3: no reply
> can someone guess why?
Please elaborate on which step it failed, was it working before applying rules, some outputs showing symptoms is usually helpful for starting troubleshooting.

**Comment by ARCHAJ** [comment]
Hi,
Thank you very much.
I have figure out that the reason for that is the following error message when running the sfc_app.py:
OperationalError: attempt to write a readonly database.
So I changed the permissions for the file nfv.sqlite and the folder sfc_app-master.
Then I closed the application and Mininet and it worked fine.
This is what I get:
mininet> h1 tracepath h5
 1?: [LOCALHOST]                                         pmtu 1500
 1:  10.0.0.3                                              4.282ms 
 2:  10.0.0.2                                              3.548ms asymm  1 
 3:  10.0.0.5                                              7.173ms reached
     Resume: pmtu 1500 hops 3 back 1 
Cordially,
Hajar
